models/src/main/protobuf/CasperMessage.proto

```diff
@@ -117,8 +117,7 @@ message BlockMetadataInternal {
 
 message Header {
   repeated bytes parentsHashList = 1; //list of parent block hashes
-  bytes postStateHash      = 2;
-  bytes deploysHash        = 3;
+  bytes deploysHash = 3;
```

Comment:
Changing the protobuf might cause the test_slash integration tests failed because the tests use the protobuf.